containers/docker/deployment/swarm/bin/clean_run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+SCRIPT_HOME=$(dirname $0)
+. ${SCRIPT_HOME}/common
+
+
```

Comment:
minor: we should output a message warning all containers will be stopped and removed

Reply:
Even more reason to output a warning message!

Reply:
I add a prompt: "This command will delete all running and stopped containers. Do you want to continue? [Y|n]"